src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas3/TimelineController.kt

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.controller.cas3
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.cas3.TimelineCas3Delegate
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ReferralHistoryNote
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.TemporaryAccommodationAssessmentEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.AssessmentService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.UserService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.cas3.DomainEventService
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer.AssessmentTransformer
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.extractEntityFromCasResult
+
+@RestController("CAS3TimelineController")
+class TimelineController(
```

Comment:
Do we think there's any value having the class exposed under a different name in spring to it's class name? How about we rename the class to `Cas3TimelineController` and drop the value from the annotation?

(on a wider note, i think we should be prefixing all casx-specific classes with the corresponding CAS code to make it easier to navigate the code)